TransformationSystem/Client/TaskManager.py

```diff
@@ -169,20 +235,34 @@ def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGrou
           transfer.addFile( trFile )
 
       oRequest.addOperation( transfer )
-      oRequest.RequestName = _requestName( transID, taskID )
-      oRequest.OwnerDN = ownerDN
-      oRequest.OwnerGroup = ownerGroup
-
-      isValid = self.requestValidator.validate( oRequest )
-      if not isValid['OK']:
-        self.log.error( "Error creating request for task", "%s %s" % ( taskID, isValid ) )
-        # This works because we loop over a copy of the keys !
-        taskDict.pop( taskID )
-        continue
+      self._addOrPop( oRequest, taskDict, transID, taskID, ownerDN, ownerGroup )
+
+  def _addOrPop( self, oRequest, taskDict, transID, taskID, ownerDN, ownerGroup ):
```

Comment:
May I just recommand a different name ? The comments are very clear but I find the name misleading, but this is just personal maybe. But reading the name, I got the feeling that adding and poping were targeted at the same entity, while if fact you add a request or pop a transformation. I would just call it something like ```_addRequestToTransformation``` or anything like that, and the poping is actually part of the error handling. 
But again, this is minor, and just a matter of personal taste I guess